### ISSUE TITLE
Continuous label as a vector

### ISSUE THREAD

**Issue by suessmann** [UBCDingXin/improved_CcGAN#1]
Hello!

Thanx for the paper, I find the proposed approach new and promising.

I have a question for the authors: say, I do not have a distinct label for each image, but rather a continuous vector of size (1, 3). Is it still possible to use this approach? In this case, a number of distinct classes is almost equal to the distinct images in the dataset.

**Comment by UBCDingXin** [comment]
Hi,

Thanks for your interest in our work. 

For your question, if my understanding is correct, you want to replace the continuous, scalar condition with a continuous, **multivariate** condition? If so, I think our approach is still applicable once you can define a reasonable metric between any two such multivariate conditions and you have an efficient way to compute it during training. However, we didn't conduct an experiment in such a scenario, but I think it worths a try.

**Comment by suessmann** [comment]
Thank you for the answer! Btw, congrats with ICLR 2021 acceptance!

Just to clarify some things: you guessed right, I meant multivariate condition. However, I still struggle with hyperparameter search. For evaluation of kappa, we need to sort the labels in ascending order, then find pairwise difference and take max (if I get it right). But in the multivariate setting I cannot yet think of a way to calculate kappa. May you suggest anything to think about in this context?

**Comment by UBCDingXin** [comment]
Thanks!

Yes, you are right. It is more difficult to compute kappa in the multivariate setting than the univariate setting. The rule of thumb for the hyper-parameter tuning in the univariate setting may not be applicable in the multivariate setting, since sorting distinct labels, as the first step of the rule of thumb, is not applicable anymore. Unfortunately, I don't have a good solution right now. As a makeshift, you may set kappa as a percentile (say 80%) of the pairwise differences (between any two distinct labels; measured by a suitable metric). Not sure whether it can work...

**Comment by suessmann** [comment]
Ok, I see. Will try to make it work, thanks for your explanation!
